### PR TITLE
Add EC2 Request

### PR DESCRIPTION
**Pull request by jgockley62**
new file:   config/prod/JKG-ec2-v5.yaml

### CODE REVIEW

File: config/prod/JKG-ec2-v5.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # (Optional) EC2 instance type, default is "t2.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.xlarge"
+  # (Optional) ID of the base AMI (supported distros: AWS linux)
+  #AMIId: "ami-0d051218cbaf3405a"      # AMI with my R packages
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  #VolumeSize: "50"
```
Comment (thomasyu888) [inline config/prod/JKG-ec2-v5.yaml]: Did you want an instance larger than 8GB?

Comment (jgockley62) [reply config/prod/JKG-ec2-v5.yaml]: Ideally I'd like to keep it small. Can I mount an s3 larger than 8GB, or is 8GB always the limit?

Comment (zaro0508) [reply config/prod/JKG-ec2-v5.yaml]: This is referring to the volume on the EC2, not S3.  Look at the comment on the previous line for default and limit: "# (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB"

Comment (zaro0508) [reply config/prod/JKG-ec2-v5.yaml]: This is referring to the volume on the EC2, not S3. Look at the comment on the previous line for default and limit: "# (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB"

**Pull request merged**